### PR TITLE
Improve Get Involved layout

### PR DESCRIPTION
**Pull request by rubenwardy**
The purpose of this PR to improve the space efficiency of this page, to make it friendlier to noobs. To do this, I've made better use of headings, columns, and icons, and removed unnecessary sections such as IRC rules

## Todo

- [ ] Add icons

## Screenshot

image

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@
   }
 }
 
-a:not(.navbar-item) {
+a:not(.navbar-item,.no-underline) {
```
Comment (Calinou) [inline _sass/_base.scss]: Add a space after the comma (if that's allowed in `not()` in CSS):

```suggestion
a:not(.navbar-item, .no-underline) {
```

File: _sass/_base.scss [hunk 2]
```diff
@@ -32,6 +32,16 @@ a:not(.navbar-item) {
   }
 }
 
+.hover-enlarge {
```
Comment (Calinou) [inline _sass/_base.scss]: Use 2-space indentation in this block (like in other CSS files).

**Comment by benrob0329** [comment]
This will probably fulfill most of your needs for icons: https://fontawesome.com/

**Comment by rubenwardy** [comment]
image

**Comment by rubenwardy** [comment]
Added screenshots, fixed alignments, fixed review comments, see OP

File: _includes/footer.html [hunk 3]
```diff
@@ -51,7 +51,8 @@ <h5 class="footer-title">Development</h5>
 
   <div class="footer-copyright">
     © 2015-{{ site.time | date: '%Y' }} The Minetest Team.
+    <a href="https://github.com/minetest/minetest.github.io">Source</a><br />
     MIT for code, CC-BY-SA 3.0 for media and content.
-    <a href="https://github.com/minetest/minetest.github.io">Source</a>
+    Fontawesome icons under <a href="https://fontawesome.com/license">CC-BY 4.0</a>.
```
Comment (Calinou) [inline _includes/footer.html]:
```suggestion
    Font Awesome icons under <a href="https://fontawesome.com/license">CC-BY 4.0</a>.
```

File: _sass/_base.scss [hunk 4]
```diff
@@ -43,3 +53,16 @@ a:not(.navbar-item) {
...
   margin-bottom: 3rem;
 }
+
+.iconlist {
+  .image {
+    width: 100px;
+    height: 120px;
+    padding: 10px 0;
+    margin: auto;
+  }
```
Comment (Calinou) [inline _sass/_base.scss]: Add a line break after `}`.

**Pull request merged**